spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -488,8 +488,9 @@ private void setupConfForPySpark(SparkConf conf) {
     }
 
     //Only one of py4j-0.9-src.zip and py4j-0.8.2.1-src.zip should exist
+    //TODO(zjffdu), this is not maintainable when new version is added.
```

Comment:
agreed. I think we should follow up with a PR to change this to search under SPARK_HOME for py4j-* instead of hard coding the filename